src/ts/uimanager.ts

```diff
@@ -902,3 +899,15 @@ export class PlayerWrapper {
   }
 }
 
+function getAllPropertyNames(target: PlayerAPI): string[] {
```

Comment:
Would it make sense to enable it for 'any-object' (in case it is needed for a different type than the `PlayerAPI`)
```suggestion
function getAllPropertyNames(target: Object): string[] {
```